app/views/content/train-to-be-a-teacher/teaching-degree-apprenticeships.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Teacher degree apprenticeships (TDAs)"
+heading: "Teacher degree apprenticeships (TDAs)"
+description: |-
+ A new route to become a teacher without a degree, a teacher degree apprenticeship (TDA) lets you work at a school while you qualify as a teacher.
+related_content:
+    Is teaching right for me? : "/is-teaching-right-for-me"
+    Train to teach if you do not have a degree : "/train-to-be-a-teacher/if-you-dont-have-a-degree"
+    Who do you want to teach? : "/is-teaching-right-for-me/who-do-you-want-to-teach"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-undergraduate-course
+navigation: 20.20
+navigation_title: Teacher degree apprenticeships (TDAs)
+navigation_description: A TDA lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
```

Comment:
```suggestion
navigation_description: An apprenticeship lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
```
TDA is very much internal jargon - we are better off using apprenticeship for understandability and also SEO?

Reply:
I changed the title of the card to say apprenticeship and spelled out TDA in full as it does also come up in search.